docs/reference/rise-primitives.md

```diff
@@ -0,0 +1,37 @@
+---
+id: rise-primitives
+title: RISE Primitives
+sidebar_label: RISE Primitives
+slug: /reference/rise-primitives
+---
+
+## Type syntax cheatsheet
+
+Type | Syntax
+-----|-------
+Function from `A` to `B` | `(A -> B)`
+Dependent function from `a : A` to `B a` | `(x : A -> B)`
+Dependent function application | `f(n)`
+Vector of `A` of size `n` | `<n>A`
+Array of `A` of size `n` | `n.A`
+Dependent array of `F : Nat -> A` of size `n` | `n..F`
+Index of size `n : Nat` | `idx[n]`
+Pair of `A` and `B` | `(A, B)`
+Dependent pair of `x : A` and `B a` | `(x : A ** B)`
+
```

Comment:
Can you add a note here that there is a convention that implicit parameters are written with a leading underscore?

Reply:
good idea

Reply:
Mhm I just realised that because we use `makeClosed` to show the implicit variable bindings these get transformed into explicit variables and thus are not prefixed with an underscore anymore. Should we make the types closed and all bindings explicit or show we leave them open and not bind implicit variables but refer to them with underscores?

Reply:
I think leaving the type open sounds good? It also reflects better what argument should really be passed when writing programs.

Reply:
True, I'll leave them open :)

Reply:
@Bastacyclop a small drawback to this is that we would not be printing out the types of these type identifiers anymore (unless we print out types together with the variable reference?)

Reply:
I guess that you mean the kinds of the identifiers? The naming convention might be enough for this. Otherwise you could use `makeClosed` to spot the free variables and then print them differently. Maybe `forall n: Nat ...`.